### PR TITLE
Add fbsimctl formula

### PR DESCRIPTION
**Pull request by lawrencelomax**
Adds a formula for [`fbsimctl`, a Command Line Interface to the `FBSimulatorControl` Framework](https://github.com/facebook/FBSimulatorControl/tree/master/fbsimctl).

- [x] Make the Formula
- [x] Update this pull request when `v0.0.1` is tagged shortly, update with the SHA of the Zip Archive of the tagged revision.

cc @bhamiltoncx @sdwilsh

### CODE REVIEW

**Comment by lawrencelomax** [comment]
Works with the tagged `v0.1.0` as well as `HEAD`.

```bash
[lawrencelomax:~/src/FBSimulatorControl] master 1 ± brew install fbsimctl
==> Installing fbsimctl from facebook/fb
==> Downloading https://github.com/facebook/FBSimulatorControl/tarball/v0.1.0
==> Downloading from https://codeload.github.com/facebook/FBSimulatorControl/legacy.tar.gz/v0.1.0
######################################################################## 100.0%
==> ./build.sh cli build /usr/local/Cellar/fbsimctl/0.1.0/libexec
🍺  /usr/local/Cellar/fbsimctl/0.1.0: 554 files, 49.3M, built in 3 minutes 37 seconds
⌂65% [lawrencelomax:~/src/FBSimulatorControl] master 3m39s ± brew remove fbsimctl
Uninstalling /usr/local/Cellar/fbsimctl/0.1.0... (554 files, 49.3M)
[lawrencelomax:~/src/FBSimulatorControl] master ± brew install fbsimctl --HEAD
==> Installing fbsimctl from facebook/fb
==> Cloning https://github.com/facebook/FBSimulatorControl.git
Updating /Users/lawrencelomax/Library/Caches/Homebrew/fbsimctl--git
==> Checking out branch master
==> ./build.sh cli build /usr/local/Cellar/fbsimctl/HEAD-60de2e0/libexec
🍺  /usr/local/Cellar/fbsimctl/HEAD-60de2e0: 554 files, 49.3M, built in 2 minutes 18 seconds
```

File: fbsimctl.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "carthage"
+  depends_on :xcode => "6.0"
+
+  def install
+    system "./build.sh", "cli", "build", "#{libexec}" 
+  end
+
+  test do
+    system "fbsimctl", "list"
+    bin.install_symlink "#{libexec}/xctool"
```
Comment (bhamiltoncx) [inline fbsimctl.rb]: Why is this in `test`?

File: fbsimctl.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Fbsimctl < Formula
+  desc "A Powerful Command Line for Managing iOS Simulators"
+  homepage "https://github.com/facebook/FBSimulatorControl/fbsimctl/README.md"
+  url "https://github.com/facebook/FBSimulatorControl/tarball/v0.1.0"
+  sha256 "8e3e74cd816185ca4328618931aff879ae0050f1740393214dfef943fcda109e"
+  head "https://github.com/facebook/FBSimulatorControl.git"
+  
+  depends_on "carthage"
+  depends_on :xcode => "6.0"
```
Comment (bhamiltoncx) [inline fbsimctl.rb]: Do we also need a <= "8.0"?

**Comment by bhamiltoncx** [comment]
Nifty, gave you some thoughts.

File: fbsimctl.rb (hunk 1 above)
Comment (Coneko) [inline fbsimctl.rb]: I don't think it should install itself as xctool, assuming this is what this means.

Comment (lawrencelomax) [inline fbsimctl.rb]: Yeah I messed this one up! Needs to be symlinked appropriately and but in the build block.

File: fbsimctl.rb (hunk 2 above)
Comment (lawrencelomax) [inline fbsimctl.rb]: 👍

**Comment by lawrencelomax** [comment]
Updated so that we will only build in Xcode 7, as the project hasn't migrated to Swift 2.3/Swift 3 yet. Fixed the relevant `test` & `install` bits mentioned in the comments.

**Comment by bhamiltoncx** [comment]
Looks good, merged.

**Comment by lawrencelomax** [comment]
❤️

**Comment by lanzelot1989** [comment]
works great, thanks (y)

**Pull request merged**